app/experimenter/experiments/constants/nimbus.py

```diff
@@ -65,6 +65,20 @@ class NimbusTargetingConfig:
     ),
 )
 
+TARGETING_HOMEPAGE_GOOGLE = NimbusTargetingConfig(
+    name="Homepage set to google.com",
+    slug="homepage_google_dot_com",
+    description="US users (en) with their Homepage set to google.com",
+    targeting=(
+        "localeLanguageCode == 'en' && "
+        "region == 'US' && "
```

Comment:
Coulda embedded an existing targeting expression like here:

https://github.com/mozilla/experimenter/blob/main/app/experimenter/experiments/constants/nimbus.py#L40

but this is also totally fine 👍

Reply:
Let's leave it as it is then 😜. Will revisit/refactor those entries once the targeting combination gets implemented.

Reply:
Yeah it's just a thought, I'm still not sure whether that's a good pattern or not.  On the one hand it could help keep things consistent and reduce duplication, on the other hand we could wind up making this big tangled mess of targeting configs that depend on each other and changing one could break everything.  So we'll just have to see how it looks the more we use it.